CHANGES.md

```diff
@@ -1,3 +1,7 @@
+#### 0.68
+
+  - Change the signature of `FAPI.backfillFromConfig` to `FAPI.backfillFromConfig(collectionConfig: CollectionConfig, adjustSearchQuery: AdjustSearchQuery, adjustItemQuery: AdjustItemQuery)`
```

Comment:
No, I meant what you are changing it from. Doesn't really matter too much! But definitely not the full signature :)